pandas/core/series.py

```diff
@@ -1327,7 +1327,9 @@ def _maybe_update_cacher(
     # ----------------------------------------------------------------------
     # Unsorted
 
-    @property
+    # error: Cannot override final attribute "_is_mixed_type" (previously declared
+    # in base class "NDFrame")
+    @property  # type: ignore[misc]
     def _is_mixed_type(self) -> bool:
```

Comment:
`NDFrame` declares `_is_mixed_type` as final! This method should not exist or the parent method should not be final.

Not sure why mypy didn't find this earlier.

Reply:
Mypy generally doesn't check unannotated methods. It should probably still check `@final` though.